prisma-schema-wasm/Cargo.toml

```diff
@@ -7,6 +7,6 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-wasm-bindgen = "=0.2.84"
+wasm-bindgen = "=0.2.87"
```

Comment:
Must match the version of the globally installed binary. CI builds prisma-schema-wasm with nix, and fresh nixpkgs have an updated version.